src/components/boardComponents/useItemActions.js

```diff
@@ -125,6 +127,92 @@ export const useItemActions = () => {
     [getSelectedItemList, batchUpdateItems, selectedItems]
   );
 
+  // Align selection to a line
+  const alignAsLine = useRecoilCallback(
+    ({ snapshot }) => async () => {
+      const selectedItemList = await getSelectedItemList(snapshot);
+
+      // Compute
+      const minMax = { min: {}, max: {} };
+      minMax.min.x = Math.min(...selectedItemList.map(({ x }) => x));
+      minMax.min.y = Math.min(...selectedItemList.map(({ y }) => y));
+      minMax.max.x = Math.max(
+        ...selectedItemList.map(
+          ({ x, id }) => x + document.getElementById(id).clientWidth
+        )
+      );
+      minMax.max.y = Math.max(
+        ...selectedItemList.map(
+          ({ y, id }) => y //+ document.getElementById(id).clientHeight
+        )
+      );
+
+      const [newX, newY] = [minMax.min.x, minMax.max.y];
+      let index = -1;
+      batchUpdateItems(selectedItems, (item) => {
+        index += 1;
+        const { clientWidth, clientHeight } = document.getElementById(item.id);
+        return {
+          ...item,
+          x: newX + index * clientWidth,
```

Comment:
Just a note, this computation works well with same sized images, but is broken with different sizes. I will accept the PR anyway as the common use case is to align card of same size.

Reply:
I had noticed. But it is still very useful, even if the cards overlap a bit. Subsequently, it would be interesting to find a cleaner solution.